pkg/chains/formats/intotoite6/intotoite6.go

```diff
@@ -84,7 +84,7 @@ func (i *InTotoIte6) generateAttestationFromTaskRun(tr *v1beta1.TaskRun) (interf
 				ID: i.builderID,
 			},
 			BuildType:   tektonID,
-			Invocation:  invocation(tr),
+			Invocation:  invocation(tr, i.logger),
```

Comment:
Instead of passing through a logger, let's just return back the error.

Reply:
Do we want the whole attestation process to fail if that's the case? That makes sense to me, just want to confirm intended behavior.